build.sbt

```diff
@@ -162,19 +172,20 @@ def playJsonTests(scalacVersion: String, includePlayVersion: String, includeScal
     // Scala 2.13 and ScalaTest 3.1 has some source code incompatibilities that require separate source directories
     case (Play_2_5, ScalaCheck_1_14) => "play25-json-tests-sc14"
     case (Play_2_7, ScalaCheck_1_14) => "play27-json-tests-sc14"
+    case (Play_2_8, ScalaCheck_1_14) => "play28-json-tests-sc14"
     case _ => "play-json-tests-common"
   }
   commonProject(id, projectPath, scalacVersion).settings(
     Test / scalacOptions -= "-deprecation",
-    
-    //Suppress semver check since we add a new artifact in v3.2.0
+
+    //Suppress semver check for new play 2.8 modules
     semVerEnforceAfterVersion := {
       (scalacVersion, includePlayVersion, includeScalaCheckVersion) match {
-        case (Scala_2_11, Play_2_5, ScalaCheck_1_14) => Some("3.3.0")
+        case (_, Play_2_8, _) => Some("4.1.0")
```

Comment:
Q: Does it make sense to make a constant for `"4.1.0"`?  I see a lot of constants and literals in here so I'm not sure.

Reply:
I don't think so because as soon as this get released it could all get removed